lib/msal-node/test/cache/TokenCache.spec.ts

```diff
@@ -246,4 +246,68 @@ describe("TokenCache tests", () => {
             }
         );
     });
+
+    it("overwriteCache should overwrite the in-memory cache with persistent cache if exists", async () => {
+        const cachePath = "./test/cache/cache-test-files/default-cache.json";
+        const beforeCacheAccess = async (context: TokenCacheContext) => {
```

Comment:
I'm wondering if we should make a CustomCache to be used in all tests, like in [samples/msal-node-samples/client-credentials-distributed-cache/src/CustomCachePlugin.ts](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-node-samples/client-credentials-distributed-cache/src/CustomCachePlugin.ts) (minus the performance stuff. Also, the cache path can be optionally passed in, otherwise it's that default file).

I would at least define this outside of the top-most describe block

Reply:
None of the tests use the same implementation of before and afterCacheAccess, so I think I'll need more explanation on what you're envisioning here

Reply:
You're right, I'll drop this.